src/app/forms/templates/general/approval-type.template.ts

```diff
@@ -1,6 +1,7 @@
+import { VehicleTypes } from '@models/vehicle-tech-record.model';
 import { FormNode, FormNodeTypes, FormNodeViewTypes } from '../../services/dynamic-form.types';
 
-export function getTypeApprovalSection(includeCoifSection: boolean = false): FormNode {
+export function getTypeApprovalSection(vehicleType: VehicleTypes): FormNode {
```

Comment:
One concept we have tried to maintain with the templates is the ability to serve them over HTTP. Therefore, we should keep the templates as JSON as much as possible.
Is there any way we could change this to two separate templates and do the combination logic in the consuming component?